docs/src/guides/wordpress/composer/migrate.md

```diff
@@ -6,70 +6,75 @@ description: |
-    
-    Run the command:
+     To use Composer, you need:
+    - A `composer.json` file listing all the dependencies needed for your project to run (WordPress itself, its plugins and themes).
+    - A `composer.lock` file listing the exact versions of all the dependencies installed on your project. Generated from the `composer.json` file, it ensures repeatable builds until you update.
+
+    To turn your repository into a Composer repository and generate those files, run the command:
 
     ```bash
     $ composer init
     ```
 
-    This will prompt you to set some metadata attributes for the project, such as its name and license information. When you get to the part about installing dependencies, type `no`, as you will add them in a later step.
+    When prompted, set metadata attributes for your project (name, license information...). When you get to the part about installing dependencies, type `no`, as you will add them later.
 
-4. **Cleanup WordPress core**
+4. **Clean up WordPress core**
 
-    Your existing installation of WordPress core is assumed to be in some subdirectory of your repository, such as `wordpress`. When we start managing WordPress with Composer in the next step, *all code in this subdirectory is overwritten*. The [previous guide](/guides/wordpress/vanilla/_index.md) instructs you to move all plugins and themes out of WordPress core and into a separate `plugins` subdirectory, so if you have not already done so consult that guide and do so now. 
+     Your existing installation of WordPress core is assumed to be in a subdirectory of your repository (often named `wordpress`). For Composer to manage WordPress, **this subdirectory needs to be overwritten**. Make sure that you move all your important data to another subdirectory. For instance, add your plugins and themes to a separate `plugins` subdirectory. 
 
-    Inspect WordPress core carefully, and ensure that their isn't any data in that subdirectory you would not like to lose specific to your site. Unless you are managing WordPress core as a [submodule](/development/submodules.md#removing-submodules), you can now remove that subdirectory 
+    Unless you are managing WordPress core as a [submodule](/development/submodules.md#removing-submodules), you can now remove the `wordpress` subdirectory: 
 
     ```bash
     $ rm -rf wordpress
     ```
 
-    Add `wordpress` to your `.gitignore`, by adding our standard WordPress `.gitignore` to the bottom of your existing file. Platform.sh's template `.gitignore` will complete the setup for Composer, but also include other files specific to Platform.sh and developing locally with  Lando. 
```

Comment:
The end seemed like too much extra information to me but need confirmation please.